### DISCUSSION TITLE
How to set up multi cluster "Install Multi-Primary" using Istio and MicroK8s cluster

### DISCUSSION THREAD

**Discussion by ngirimana** [istio/istio#51632] [Q&A]
Hi everyone,
I want to set up multi-cluster kubernetes cluster using 2 microK8s cluster with help of istio. I have done all steps in documentation but still I can get those clusters connected so that I can access the pod from one cluster in other. If some one has help for me, I will appreciate it

**Comment by rootsongjc** [comment]
Can you elaborate on what problem you encountered?

**Reply by ngirimana** [reply]
@rootsongjc I am setting up cluster mesh with istio and two microk8s clusters but those clusters can communicate to each other. How can I resolve that issue?

**Reply by rootsongjc** [reply]
Did you follow the instructions in this document? If so, the services in the two clusters should be able to access each other via the east-west gateway https://istio.io/latest/docs/setup/install/multicluster/multi-primary_multi-network/

**Reply by ngirimana** [reply]
yes

**Reply by rootsongjc** [reply]
I am writing a blog about how to deploy a multi-cluster and multi-mesh Istio. There are several main difficulties:
- Assign identities and authentication to services across multiple clusters
- Multi-cluster service registration and discovery
- DNS resolution
I will complete this blog in two weeks.

**Reply by rootsongjc** [reply]
[A Definitive Guide to Cross-Cluster Seamless Access in Multicluster Istio Service Mesh](https://jimmysong.io/en/blog/seamless-cross-cluster-access-istio/), this blog maybe helpful for you.
